docs/codeql/codeql-for-visual-studio-code/analyzing-your-projects.rst

```diff
@@ -42,14 +42,46 @@ Downloading a database from GitHub
 
 .. include:: ../reusables/download-github-database.rst
 
+.. _filtering-databases-and-queries-by-language:
+
+Filtering databases and queries by language
+-------------------------------------------
+
+Optionally, to see databases containing a specific language and queries written for that language, you can apply a language filter using the language selector.
+
+#. To see available language filters, in the sidebar, click the **Language** title bar.
+#. Hover over the language filter you would like to apply, then click **Select**.
+
+   .. image:: ../images/codeql-for-visual-studio-code/choose-language-filter.png
+      :width: 350
+      :alt: Screenshot of the language selector. The "Select" button for a language filter is outlined in dark orange.
+
+Creating a query
+------------------------
+
+You can generate a query template for a specific language from the queries panel, then write your own code to quickly create a custom query.
+
+#. In the sidebar, hover over the **Queries** title bar, then click the **Create query** icon.
+
+   .. image:: ../images/codeql-for-visual-studio-code/create-query-icon.png
+      :width: 350
+      :alt: Screenshot of the queries panel. The "Create query" icon is outlined in dark orange.
+
+#. In the Command Palette, select the target language for your query. Selecting a language will autogenerate a folder labeled "codeql-custom-queries-LANGUAGE", which will contain a query template labeled "example.ql".
```

Comment:
> folder labeled "codeql-custom-queries-LANGUAGE"

Minor: I could be wrong but I don't think we use this pattern (placeholder) in the CodeQL docs. I did search for a bit and couldn't find any examples so perhaps it's just not my memory that's telling me that 😁 

Even more minor, maybe worth using backticks (monospacing) for the example names

Reply:
Ah, great points! I'll definitely use backticks, and I'll switch over to a placeholder with angle brackets. I think you're right; we definitely chose to continue using that formatting for the CodeQL CLI docs, and I found an area of the CodeQL for VS Code docs that uses that formatting 👍🏼 thanks for catching those!